sklearn/metrics/_regression.py

```diff
@@ -456,21 +463,24 @@ def mean_squared_error(
     >>> y_pred = [2.5, 0.0, 2, 8]
     >>> mean_squared_error(y_true, y_pred)
     0.375
-    >>> y_true = [3, -0.5, 2, 7]
-    >>> y_pred = [2.5, 0.0, 2, 8]
-    >>> mean_squared_error(y_true, y_pred, squared=False)
-    0.612...
     >>> y_true = [[0.5, 1],[-1, 1],[7, -6]]
     >>> y_pred = [[0, 2],[-1, 2],[8, -5]]
     >>> mean_squared_error(y_true, y_pred)
     0.708...
-    >>> mean_squared_error(y_true, y_pred, squared=False)
-    0.822...
     >>> mean_squared_error(y_true, y_pred, multioutput='raw_values')
     array([0.41666667, 1.        ])
     >>> mean_squared_error(y_true, y_pred, multioutput=[0.3, 0.7])
     0.825...
     """
+    if squared != 'deprecated':
+        warnings.warn("'squared' is deprecated in version 1.3.0 and "
+                      "will be removed in 1.3.2. To calculate the "
+                      "root mean squared error, use the function"
+                      "'root_mean_squared_error'.",
```

Comment:
1.3 is already released, so we need to target 1.4. 1.x.x are bug fix releases, we don't make api changes in bug fix releases, so the deprecation cycle has to span 2 minor releases, i.e. end in 1.6
```suggestion
        warnings.warn("'squared' is deprecated in version 1.4 and "
                      "will be removed in 1.6 To calculate the "
                      "root mean squared error, use the function"
                      "'root_mean_squared_error'.",
```